sharpd/sharp_vty.c

```diff
@@ -63,23 +63,34 @@ DEFPY(watch_nexthop_v6, watch_nexthop_v6_cmd,
 	p.family = AF_INET6;
 
 	sharp_nh_tracker_get(&p);
-	sharp_zebra_nexthop_watch(&p, type_import, true, !!connected);
+	sharp_zebra_nexthop_watch(&p, VRF_DEFAULT, type_import,
+				  true, !!connected);
 
 	return CMD_SUCCESS;
 }
 
 DEFPY(watch_nexthop_v4, watch_nexthop_v4_cmd,
-      "sharp watch <nexthop$n|import$import> A.B.C.D$nhop [connected$connected]",
+      "sharp watch [vrf NAME$name] <nexthop$n|import$import> A.B.C.D$nhop [connected$connected]",
```

Comment:
Any reason for not adding the VRF parameter to the `watch_nexthop_v6` command (the IPv6 counterpart of this one)?

Reply:
no a total miss, I'll fix up